pkg/reconciler/v1alpha1/eventlistener/controller.go

```diff
@@ -64,5 +68,17 @@ func NewController(ctx context.Context, cmw configmap.Watcher) *controller.Impl
 		DeleteFunc: impl.Enqueue,
 	})
 
+	deploymentInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    impl.EnqueueControllerOf,
```

Comment:
This is using the same implementation as the EventListener? If that is the case, then any deployment/service created would trigger the Reconcile loop, but only the generated/children deployment/service should be reconciled?

Reply:
I suppose it works since the top of the reconciler has the following logic: 
```go
// Get the Event Listener resource with this namespace/name
	original, err := c.eventListenerLister.EventListeners(namespace).Get(name)
	if errors.IsNotFound(err) {
		// The resource no longer exists, in which case we stop processing.
		c.Logger.Infof("EventListener %q in work queue no longer exists", key)
		return nil
	} else if err != nil {
		c.Logger.Errorf("Error retreiving EventListener %q: %s", name, err)
		return err
	}
```

Reply:
Thanks for comments.
There is a mistake here, should use `cache.FilteringResourceEventHandler`, that the `deployment/svc` with incorrect owner will not go into the queue.
I have correct it, thanks again.